packages/smart-vaults/src/signers/MultiSigner.sol

```diff
@@ -17,7 +17,7 @@ struct MultiSignerStorage {
     mapping(uint8 => Signer) signers;
```

Comment:
does it make more sense to use Signers[type(uint8).max] here instead of a mapping? i know that came up before but i can't remember why we decided against it

Reply:
I haven't been able to think of strong reasons to go one way or the other. A mapping will be very slightly more expensive.

Reply:
imo fixed length array is "right by construction" vs mapping needs manual checks for whether index is "in bounds". that alone causes me to lean toward the fixed array (unless we wanted flexibility to extend the # of signers later which...... i dont believe we do)